config/routes.rb

```diff
@@ -5,7 +5,7 @@
 
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/216604868-aca0f951-bf91-4471-8213-458ef0644dc6.png)

- [ ] Instead of **Server** kindly rename it to **Framework** because you are also making front end part so kindly rename **Server** to **Framework** to make your readme professional :+1: